dashboard/db/migrate/20190214005506_create_pd_scholarship_infos.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePdScholarshipInfos < ActiveRecord::Migration[5.0]
```

Comment:
It's interesting that this kind of class name starts with a verb. Ruby is strange :)

Reply:
yeah - migrations (which update the db schema) are a special thing, sort of like mailers.